### ISSUE TITLE
Install

### ISSUE THREAD

**Issue by williamswebworks** [gt4w-consultoria/react-native-esri#4]
Is this working for React Native still? If so...are their install steps?

**Comment by teofilosalgado** [comment]
Yes sir, it is still working. The install steps are simple, just add this repository as an dependency in your package.json like so:
`react-native-esri: [this repo URL]`

Then, for iOS just run pod install and you are good to go. For Android it should work automatically. If you have any issues or questions feel free to ask!

**Comment by williamswebworks** [comment]
I didn’t see any example of the library using an arcGIS web map. Do you happen to have one?

Thanks

**Comment by teofilosalgado** [comment]
Yeah, sure. Go to the `develop` branch and run this library as a regular react native app, you will be able to play around with the included example. The source code is in `App.js` file.

**Comment by williamswebworks** [comment]
Ok thanks that is helpful…Does it work with a web map?

**Comment by teofilosalgado** [comment]
Could you be a little bit more specific? It would be very helpful if you elaborate more about this web map, if possible 😅. 
With that said, this library was designed to work only with ArcGIS Map Services or Features, you might need to check if your service fits these categories.

**Comment by williamswebworks** [comment]
There are maps that you can create with arcGIS online that are made with feature maps - ex. http://www.arcgis.com/home/item.html?id=6fbefe3fc8b6486290bae54378fd9d01

Other libraries have been created that allow you to embed a webmap via ID or url. I was trying to determine if this library allows this also

**Comment by teofilosalgado** [comment]
Sorry to say that but we currently do not support AGOL maps 😕. Although it's not that hard to be implemented, to be honest, you just need to instantiate your `AGSMap` from an `AGSPortalItem` like so: [https://developers.arcgis.com/ios/latest/swift/guide/access-content.htm](https://developers.arcgis.com/ios/latest/swift/guide/access-content.htm). You may fork this or any other ArcGIS map wrapper for react native and add this functionality, if you are comfortable with doing it.

**Comment by williamswebworks** [comment]
I need it in react native
